tests/framework/installer/test_data.go

```diff
@@ -29,6 +29,7 @@ kind: StorageClass
 metadata:
    name: ` + storageClassName + `
 provisioner: ceph.rook.io/block
+reclaimPolicy: ` + reclaimPolicy + `
```

Comment:
In addition to passing the different reclaim policies, do you verify if the reclaim policy outcome was correct?

Reply:
@travisn this is still `work-in-progress`. I was wondering how do I fetch a `PV` instance to assert whether they came up with the proper `reclaimPolicy`. I was looking into https://github.com/rook/rook/blob/master/tests/framework/utils/k8s_helper.go to find a suitable method, however it does not seem to have what I want.

Perhaps, I can add a new function similar to https://github.com/rook/rook/blob/master/tests/framework/utils/k8s_helper.go#L626 to return me a `PersistentVolume` against which I can assert the `reclaimPolicy`.

@travisn Let me know what do you think? Do you know of a better way to do these assertions than what I proposed?

Reply:
@rtnpro There are a couple different ways that may be sufficient. Verifying the setting on the PV should work as you suggested. Another thought is to add a check after step 10 of the test where the pvc is deleted to check if the ceph block image was deleted or if it still exists. The integration tests can run ceph commands in the toolbox. If you want to try this approach i'll dig up an example of how to run the ceph command.

Reply:
@travisn I like the approach you suggested. It's in line with the current way the integration tests. It will be great if you could show me an example.

Reply:
The [test framework clients](https://github.com/rook/rook/tree/master/tests/framework/clients) provide some helpers that either may give you what you need, or you can follow their pattern. For example, the [block list](https://github.com/rook/rook/blob/master/tests/framework/clients/block.go#L81) helper will get all the block images in the cluster. We would expect the block image to be deleted if the policy is reclaim. Or if the policy is to retain, the block image should still exist. 

From the block integration test, perhaps you could call `helper.BlockClient.List()` before the delete, then call it again after the delete and see if the image was deleted. You might have to wait in a retry loop in case there is a delay to delete it.